src/typings/react-feature-toggles/index.d.ts

```diff
@@ -0,0 +1,63 @@
+declare module '@paralleldrive/react-feature-toggles' {
+  function Feature(inactiveComponent: any, activeComponent: any, name: string, children?: any): JSX.Element
+  export { Feature }
+
+  function FeatureToggles({
+    features,
+  }?: {
+    readonly features?: ReadonlyArray<string>
+    readonly children?: any
+  }): JSX.Element
+  export { FeatureToggles }
+
+  interface Feature {
+    readonly name: string
+    readonly isActive: boolean
+    readonly dependencies?: ReadonlyArray<string>
+  }
+
+  function getActiveFeatures(features: ReadonlyArray<Feature>): ReadonlyArray<string>
+  export { getActiveFeatures }
+
+  interface Query {
+    readonly ft: string
+  }
+  function getQueryFeatures(query?: Query): ReadonlyArray<string>
+  export { getQueryFeatures }
+
+  function mergeFeatures(
+    currentFeatures: ReadonlyArray<string>,
+    restOfFeatures: ReadonlyArray<string>
+  ): ReadonlyArray<string>
+  export { mergeFeatures }
+
+  function removeFeatures(
+    currentFeatures: ReadonlyArray<string>,
+    features: ReadonlyArray<string>
+  ): ReadonlyArray<string>
+  export { removeFeatures }
+
+  function isActive(name: string, currentFeatures: ReadonlyArray<string>): boolean
+  export { isActive }
+
+  interface Req {
+    readonly query: Query
+  }
+
+  function getReqQueryFeatures(req?: Req): ReadonlyArray<string>
+  export { getReqQueryFeatures }
+
+  function getBrowserQueryFeatures(search?: string): ReadonlyArray<string>
```

Comment:
This was not working for some reason. I am having trouble with it.

Reply:
What problem are you having with this one?

Reply:
I'm honestly not really sure what is going on. I thought that I had the types correct and everything, but whenever I'm trying to test to see if it actually works it says that it isn't a function. ```TypeError: react_feature_toggles_1.getBrowserQueryFeatures is not a function```

Reply:
This was a problem with react-feature-toggles, I fixed it over there.